SocialCareCaseViewerApi/V1/Boundary/Requests/DeleteCaseSubmissionRequest.cs

```diff
@@ -0,0 +1,30 @@
+using FluentValidation;
+using System.Text.Json.Serialization;
+
```

Comment:
I think we would benefit if we #enable nullable here

Reply:
yes, that should be the default setup. Added